src/js/models/maps/Map.js

```diff
@@ -31,7 +32,13 @@ define([
        * @property {MapConfig#MapAssetConfig[]} [layers] - A collection of
        * imagery, tiles, vector data, etc. to display on the map. Layers wil be
        * displayed in the order they appear. The array of the layer
-       * MapAssetConfigs are passed to a {@link MapAssets} collection.
+       * MapAssetConfigs are passed to a {@link MapAssets} collection. When layerCategories
+       * exist, this property will be ignored.
+       * @property {MapConfig#MapAssetConfig[]} [layerCategories] - A collection of
```

Comment:
@robyngit Given the new field introduced here, I imagine we need an update to docs/docs/MapConfig.html? Blame suggests these docs are autogenerated before release though- I take that it's a noop in this PR?

Reply:
Yes! Thank you for thinking of documentation. :) All of the [API docs](https://nceas.github.io/metacatui/docs/) are auto-generated from the JSdocs in all of the JS files. That's why it's so important to add/update JSdocs for any new or changed methods or properties. These docs are built at each release. If you'd like to test the docs your self, see this [README](https://github.com/NCEAS/metacatui/tree/main/docs#readme). (But please don't commit the changes from your new doc build, it just complicates the git & PR history.)

It looks like you added the new option to the MapConfig namespace so the matching documentation will be generated at release time! 🎉